src/applications/facility-locator/components/ResultsList.jsx

```diff
@@ -1,14 +1,33 @@
 /* eslint-disable arrow-body-style */
 import React, { Component } from 'react';
+import _ from 'lodash';
 import PropTypes from 'prop-types';
 import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
 import { updateSearchQuery, searchWithBounds } from '../actions';
 import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
 import SearchResult from './SearchResult';
 import Pagination from '@department-of-veterans-affairs/formation-react/Pagination';
+import { facilityTypes } from '../config';
 
 class ResultsList extends Component {
+  constructor(props) {
+    super(props);
+    this.searchResultTitle = React.createRef();
+  }
+  shouldComponentUpdate(nextProps) {
+    return (
+      nextProps.results !== this.props.results ||
+      nextProps.inProgress !== this.props.inProgress
+    );
+  }
+
+  componentDidUpdate() {
+    if (this.searchResultTitle.current) {
+      this.searchResultTitle.current.focus();
```

Comment:
this will be focused if the page is loading results- e.g. if you navigate to a url that triggers a search. This seems like the logical behavior but it's a little odd.